.github/workflows/e2e-tests-pr.yml

```diff
@@ -7,13 +7,9 @@ on:
       - 'master'
 
 jobs:
-  runTests:
-    name: 'Run E2E tests'
+  cancel-previous-runs:
     runs-on: ubuntu-latest
-    if: ${{ !contains(github.event.pull_request.labels.*.name, 'skip-e2e') }}
-
-    timeout-minutes: 30
-
+    name: 'Cancel previous e2e test runs'
```

Comment:
maybe we should stop doing this.. otherwise cypress run hangs and our dashboard looks weird

Reply:
Commented it out for now